draft-ietf-quic-transport.md

```diff
@@ -5061,7 +5060,7 @@ The first byte in the stream has an offset of 0.  The largest offset delivered
 on a stream - the sum of the offset and data length - cannot exceed 2^62-1, as
 it is not possible to provide flow control credit for that data.  Receipt of a
 frame that exceeds this limit will be treated as a connection error of type
-FLOW_CONTROL_ERROR.
+FRAME_ENCODING_ERROR.
```

Comment:
This creates an extra check that some implementations might not bother to implement.  Treating this as a flow control error is also fine.  I note that this doesn't use a MUST either...